pandas/tests/io/sas/test_sas7bdat.py

```diff
@@ -96,7 +96,7 @@ def test_path_localpath(self, dirpath, data_test_ix):
     @pytest.mark.parametrize("chunksize", (3, 5, 10, 11))
     @pytest.mark.parametrize("k", range(1, 17))
     def test_iterator_loop(self, dirpath, k, chunksize):
-        # github #13654
+        # GitHub issue #13654
```

Comment:
I'd prefer keeping this as is to keep git blame. This is an internal comment, so the spelling does not matter much